modal.js

```diff
@@ -0,0 +1,67 @@
+const modal = document.getElementById("projectDetails");
+
+// Get the button that opens the modal
+const btn = document.getElementById("details");
+
+// Get the <span> element that closes the modal
+const span = document.getElementsByClassName("closed")[0];
+
+// When the user clicks on the button, open the modal
+btn.onclick = function() {
+    modal.style.display = "block";
+}
+
+// When the user clicks on <span> (x), close the modal
+span.onclick = function() {
+    modal.style.display = "none";
+}
+
+// When the user clicks anywhere outside of the modal, close it
+window.onclick = function(event) {
+    if (event.target == modal) {
+    modal.style.display = "none";
+    }
+} 
+
+
+//Project details JS to Html
+const pdetails = {
+    title: 'Project name goes here',
+    text1: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi',
+    text2: 'Ut aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.',
+    slideshow: './images/slideshow.svg',
+    languages: {
+    lan1: 'HTML/CSS',
+    lan2: 'Ruby on Rails',
+    lan3: 'JavaScript',
+    },
```

Comment:
- [ ] Kindly add the modal popup for all the projects shown in the video attached. Make sure to add a modal popup to all buttons named "See this project".
- [ ] Kindly add a link to the live version, and link to the project source code to the objects array.

https://user-images.githubusercontent.com/70269507/163392941-bc4dbf09-38a3-410f-b683-f985842f38bc.mov